mongodb/spring-cloud-starter-stream-source-mongodb/src/main/java/org/springframework/cloud/stream/app/mongodb/source/MongoDBSourceConfiguration.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2016 the original author or authors.
```

Comment:
The Copyright doesn't follow with our standards.
You can copy/paste it from existing classes, e.g. `JdbcSourceConfiguration`